python/pyspark/pandas/groupby.py

```diff
@@ -910,7 +910,7 @@ def sum(self, numeric_only: Optional[bool] = True, min_count: int = 0) -> FrameL
 
```

Comment:
I think you gotta fix the log above too since not we support strings too?

Reply:
Yeah, we should update. Thanks for catching this out!